frontend/cypress/pages/DashBoard.js

```diff
@@ -1,81 +1,23 @@
 class DashBoardPage {
-  addOrder(Program) {
```

Comment:
@Bahati308  why ddnt you just update the selctors instead of removing this entire test case.